rooms/windyTylne.cpp

```diff
@@ -71,59 +70,66 @@ bool lift_repairing_game(GameData& gameData) {  // TODO(seqre): Add minigame
   return false;
 }
 
-void choose_room_text(GameData& gameData) {
-  std::cout << "Co robisz?" << std::endl;
-  wait_ms(200);
-  std::cout << (gameData.winda_repaired
-                    ? "1. Korzystam z windy i zjeżdżam na parter"
-                    : "1. Podchodzę do windy, by zobaczyć czemu nie działa")
-            << std::endl
-            << "2. " << (gameData.zaionc ? "Biegnę" : "Idę")
-            << " do windy z przodu budynku" << std::endl
-            << "3. " << (gameData.zaionc ? "Biegnę" : "Idę") << " do biblioteki"
-            << std::endl
-            << "4. " << (gameData.zaionc ? "Biegnę" : "Idę")
-            << " do garażu (i robię przyps)" << std::endl
-            << "5. Wracam " << (gameData.zaionc ? "biegiem" : "")
-            << "na korytarz" << std::endl;
+template <typename S>
```

Comment:
This needs to be moved to `utils`.

Reply:
To `main.hpp` but ok

Reply:
Yep - this shouldn't be a in a code file nonetheless.